urlparse4/__init__.py

```diff
@@ -1,8 +1,11 @@
 # https://github.com/python/cpython/blob/2.7/Lib/urlparse.py
 
-from urlparse import *
+import six
+
+from six.moves.urllib.parse import urlparse, urlsplit, urljoin
```

Comment:
I think the idea behind ``from urlparse import *`` was to provide all functions from urlparse module, and then override some of them below, so this should be ``from six.moves.urllib.parse import *`` now

Reply:
I got this error when importing all the modules of urllib:
```
NameError: name 'urlsplit' is not defined
```

Reply:
Should I import all the functions in the library manually?